src/components/ReplayableVideoAsset.vue

```diff
@@ -9,7 +9,18 @@
 -->
 
 <template>
-  <div class="video-replay-container">
+  <div
+    class="video-replay-container"
+    role="group"
+    :aria-roledescription="$t('video.title')"
+    :aria-labelledby="!showsDefaultControls ? ariaLabelledByContainer : null"
+  >
+    <span
+      id="custom-controls"
```

Comment:
Is this `id` necessary? It will end getting duplicated when there are multiple replayable videos on a page, which we should try to avoid if possible.

Reply:
True. I created unique ids for these elements: https://github.com/apple/swift-docc-render/pull/807/commits/813aa3fd78c51b8488b33f1b5fda90fe80f3d38d

Reply:
I also deleted delete dots from video identifiers. Because dots may cause issues in the DOM https://github.com/apple/swift-docc-render/pull/807/commits/b2b5d3ee8db097f88d5ba8d8d98ceb727a54da73